drivers/sx127x/include/sx127x_registers.h

```diff
@@ -0,0 +1,1710 @@
+#define SX127X_REG_LR_MODEMCONFIG2                                   0x1E
+#define SX127X_REG_LR_SYMBTIMEOUTLSB                                 0x1F
+#define SX127X_REG_LR_PREAMBLEMSB                                    0x20
+#define SX127X_REG_LR_PREAMBLELSB                                    0x21
+#define SX127X_REG_LR_PAYLOADLENGTH                                  0x22
+#define SX127X_REG_LR_PAYLOADMAXLENGTH                               0x23
+#define SX127X_REG_LR_HOPPERIOD                                      0x24
+#define SX127X_REG_LR_FIFORXBYTEADDR                                 0x25
+#define SX127X_REG_LR_MODEMCONFIG3                                   0x26
+#define SX127X_REG_LR_FEIMSB                                         0x28
+#define SX127X_REG_LR_FEIMID                                         0x29
+#define SX127X_REG_LR_FEILSB                                         0x2A
+#define SX127X_REG_LR_RSSIWIDEBAND                                   0x2C
+#define SX127X_REG_LR_TEST2F                                         0x2F
+#define SX127X_REG_LR_TEST30                                         0x30
+#define SX127X_REG_LR_DETECTOPTIMIZE                                 0x31
+#define SX127X_REG_LR_INVERTIQ                                       0x33
+#define SX127X_REG_LR_TEST36                                         0x36
+#define SX127X_REG_LR_DETECTIONTHRESHOLD                             0x37
+#define SX127X_REG_LR_SYNCWORD                                       0x39
+#define SX127X_REG_LR_TEST3A                                         0x3A
+#define SX127X_REG_LR_INVERTIQ2                                      0x3B
+
+/** I/O settings */
+#define SX127X_REG_LR_DIOMAPPING1                                    0x40
+#define SX127X_REG_LR_DIOMAPPING2                                    0x41
+
+/** Version */
+#define SX127X_REG_LR_VERSION                                        0x42
+
+/** Additional settings */
+#define SX127X_REG_LR_PLLHOP                                         0x44
+#define SX127X_REG_LR_TCXO                                           0x4B
+#define SX127X_REG_LR_PADAC                                          0x4D
+#define SX127X_REG_LR_FORMERTEMP                                     0x5B
+#define SX127X_REG_LR_BITRATEFRAC                                    0x5D
+#define SX127X_REG_LR_AGCREF                                         0x61
+#define SX127X_REG_LR_AGCTHRESH1                                     0x62
+#define SX127X_REG_LR_AGCTHRESH2                                     0x63
+#define SX127X_REG_LR_AGCTHRESH3                                     0x64
+#define SX127X_REG_LR_PLL                                            0x70
+
+/**
+ *	Modem settings for random number generator
+ */
+#define SX127X_RNG_REG_MODEM_CONFIG1                                 0x0A
+#define SX127X_RNG_REG_MODEM_CONFIG2                                 0x70
+
+
+/**
+ * SX127X LoRa bits control definition
+ */
+
+/**
+ * RegFifo
```

Comment:
CamelCase is widely used in Semtech's reference design code from what that driver's sources and definition files was mostly inherited.